client/settings/express-checkout/style.scss

```diff
@@ -12,7 +12,7 @@
 			.loadable-checkbox__icon .gridicons-notice-outline {
 				fill: #f0b849;
 				margin-bottom: -5px;
-				margin-right: 16px;
```

Comment:
I noticed there was a bit of dancing when checking/unchecking Link, so I am piggybacking on this PR to fix it

![2024-08-15 12 14 54](https://github.com/user-attachments/assets/a8a3cf7a-fc55-42e0-8af8-6be36b66a9b6)